### ISSUE TITLE
GINConv does not support prelu activation.

### ISSUE THREAD

**Issue by ChenYizhu97** [danielegrattarola/spektral#372]
Hi, I find that the GINConv doesn't support the prelu activation for its hidden mlp, because it define the activation inside the keras.Dense layer, which doesn't support prelu natively.

    self.mlp = Sequential()
        for channels in self.mlp_hidden:
            self.mlp.add(Dense(channels, self.mlp_activation, **layer_kwargs))
            if self.mlp_batchnorm:
                self.mlp.add(BatchNormalization())
        self.mlp.add(
            Dense(
                self.channels, self.activation, use_bias=self.use_bias, **layer_kwargs
            )
        )

However, in the codes of GeneralGNN, exactly the codes of MLP class, the is a workaround for this issue by defining the activation as a layer.

    self.mlp = Sequential()
        for i in range(layers):
            # Linear
            self.mlp.add(Dense(hidden if i < layers - 1 else output))
            # Batch norm
            if self.batch_norm:
                self.mlp.add(BatchNormalization())
            # Dropout
            self.mlp.add(Dropout(self.dropout_rate))
            # Activation
            self.mlp.add(get_act(activation if i < layers - 1 else final_activation))

in which get_act is defined by:

    def get_act(identifier):
        if identifier == "prelu":
            return PReLU()
        else:
            return Activation(identifier)

Will you consider to modify the GINConv layer in this way to support more activation functions in keras?

**Comment by danielegrattarola** [comment]
That workaround in GeneralGNN is non-standard behaviour for Keras and it's also not fully functioning (no parameters can be given to the PReLU activation), so it's not a good idea to duplicate it in other parts of the library.

If you want that functionality, it's better to extend `GINConv` and override the `build` method yourself.

Cheers
